aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/transfer/TransferTable.kt

```diff
@@ -213,6 +216,9 @@ internal class TransferTable {
             if (TABLE_VERSION_8 in (oldVersion + 1)..newVersion) {
                 addVersion8Columns(database)
             }
+            if (TABLE_VERSION_9 in (oldVersion + 1)..newVersion) {
```

Comment:
Can we get a test on this migration? If not in code, at least confirmation v8 db with existing entries was observed to migrate successfully and set default values of existing rows to 0

Reply:
![image](https://user-images.githubusercontent.com/83975678/223479075-1e97a356-cddc-40b1-85ab-e96a2bd6e6c1.png)